plugins-compat-tester-model/src/main/java/org/jenkins/tools/test/model/PomData.java

```diff
@@ -45,8 +46,9 @@
     private String connectionUrl;
     private List<String> warningMessages = new ArrayList<String>();
 
-    public PomData(String artifactId, @CheckForNull String packaging, String connectionUrl, @CheckForNull MavenCoordinates parent){
+    public PomData(String artifactId, @CheckForNull String packaging, String connectionUrl, @CheckForNull MavenCoordinates parent, String groupId){
```

Comment:
Retain binary compatibility?

Reply:
I don't think it's used outside? But worth to verify that.

Reply:
It is only used in this repo, so I guess it is fine.

Reply:
We can live with that